### PR TITLE
spirv_headers is deprecated

### PR DESCRIPTION
**Pull request by Jasper-Bekkers**
Because of some versioning issues / semver incompatibility we've continued the development of `spirv_headers` as a seperate crate called `spirv`.

### CODE REVIEW

File: crates/spirv_headers/RUSTSEC-0000-0000 [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[advisory]
```
Comment (tarcieri) [inline crates/spirv_headers/RUSTSEC-0000-0000:1]:
```suggestion
```toml
[advisory]
```

**Comment by tarcieri** [comment]
File needs to end with `.md`

**Comment by Jasper-Bekkers** [comment]
Done :-).

**Comment by tarcieri** [comment]
Thank you!

**Pull request merged**